.github/pull_request_template.md

```
@@ -45,6 +41,19 @@ Fixes #<!--Add related issue number here.-->
 
 <!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
 
+## Checklist
+
+### CodeRabbit AI Review
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)